spec/tags/language/hash_tags.txt

```diff
@@ -1,2 +1,3 @@
 fails:The ** operator makes a caller-side copy when calling a method taking a positional Hash
 fails:Hash literal checks duplicated keys on initialization
+fails:Hash literal checks duplicated float keys on initialization
```

Comment:
This seems like an interesting failure...

Reply:
Indeed. We'll have to look at that in more detail when we bump to Ruby 3.1. Just to avoid any confusion, it's a Ruby 3.1 spec. We're only running it now because I enabled the 3.1 `Hash` specs to test the punning syntax.

Reply:
The failure is just a missing warning so I think it's not really important at this point.